### PR TITLE
Update `feedparser`

### LINKED ISSUE #4991: Update feedparser
Update the `feedparser` library:
https://github.com/kurtmckee/feedparser/tree/develop

Diff from current vendored commit ([`f1dd1bb923ebfe6482fc2521c1f150b4032289ec`](https://github.com/pymedusa/Medusa/blob/0e800bfd9e9b89f2c529dcb9b7a677b5e4d7c6e7/requirements.txt#L15)):
https://github.com/kurtmckee/feedparser/compare/f1dd1bb923ebfe6482fc2521c1f150b4032289ec...develop

### PR DESCRIPTION
**Pull request by sharkykh**
Update `feedparser` to commit 2b11c8028321ed43cbaf313f83b0c94820143d66
Closes #4991

### CODE REVIEW

**Comment by p0psicles** [comment]
Update readme?

**Comment by sharkykh** [comment]
Hmm... it doesn't look like it changes anything for the user experience.
Unlike the `guessit` update, for example.

**Comment by p0psicles** [comment]
Shouldnt we update the ext/readme. Md?

File: ext/readme.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 :: | `diskcache` | [2.9.0](https://pypi.org/project/diskcache/2.9.0/) | `imdbpie` | -
 :: | `dogpile.cache` | [0.6.7](https://pypi.org/project/dogpile.cache/0.6.7/) | **`medusa`**, `subliminal` | -
 :: | `enzyme` | pymedusa/[665cf69](https://github.com/pymedusa/enzyme/tree/665cf6948aab1c249dcc99bd9624a81d17b3302a) | `knowit`, `subliminal` | -
-:: | `feedparser` | [f1dd1bb](https://github.com/kurtmckee/feedparser/tree/f1dd1bb923ebfe6482fc2521c1f150b4032289ec) | **`medusa`** | -
+:: | `feedparser` | [2b11c80](https://github.com/kurtmckee/feedparser/tree/2b11c8028321ed43cbaf313f83b0c94820143d66) | **`medusa`** | Requires `sgmllib3k` on Python 3
```
Comment (sharkykh) [inline ext/readme.md]: Done...

File: requirements.txt [hunk 2]
```diff
@@ -12,7 +12,7 @@ contextlib2==0.5.5 ; python_version < '3.5'
 dirtyjson==1.0.7
 dogpile.cache==0.6.7
 git+https://github.com/pymedusa/enzyme.git@665cf6948aab1c249dcc99bd9624a81d17b3302a#egg=enzyme
-git+https://github.com/kurtmckee/feedparser.git@f1dd1bb923ebfe6482fc2521c1f150b4032289ec#egg=feedparser
+git+https://github.com/kurtmckee/feedparser.git@2b11c8028321ed43cbaf313f83b0c94820143d66#egg=feedparser
```
Comment (sharkykh) [inline requirements.txt]: ...and done :smile:

**Pull request merged**